buildSrc/src/main/java/com/osacky/flank/gradle/FlankGradleExtension.kt

```diff
@@ -55,51 +57,51 @@ open class FlankGradleExtension @Inject constructor(objects: ObjectFactory) : Fl
   @get:Optional
   val instrumentationApk: Property<String> = objects.property()
 
-  override var directoriesToPull: List<String> = emptyList()
+  override val directoriesToPull: ListProperty<String> = objects.listProperty()
 
-  override var filesToDownload: List<String> = emptyList()
+  override val filesToDownload: ListProperty<String> = objects.listProperty()
 
-  override var environmentVariables: Map<String, String> = emptyMap()
+  override val environmentVariables: MapProperty<String, String> = objects.mapProperty()
 
-  override var recordVideo: Boolean = true
+  override val recordVideo: Property<Boolean> = objects.property<Boolean>().convention(true)
 
-  override var performanceMetrics: Boolean = true
+  override val performanceMetrics: Property<Boolean> = objects.property<Boolean>().convention(true)
 
-  override var resultsBucket: String? = null
+  override val resultsBucket: Property<String> = objects.property()
 
-  override var keepFilePath: Boolean = false
+  override val keepFilePath: Property<Boolean> = objects.property<Boolean>().convention(false)
 
-  override var resultsDir: String? = null
+  override val resultsDir: Property<String> = objects.property()
 
   override val additionalTestApks: ListProperty<String> = objects.listProperty()
 
   override val runTimeout: Property<String> = objects.property()
 
-  override val ignoreFailedTests: Property<Boolean> = objects.property()
+  override val ignoreFailedTests: Property<Boolean> = objects.property<Boolean>().convention(false)
```

Comment:
why was this changed to false here?

Reply:
Oh...again, unintentionally....There were lots of changes and got a bit lost 😅  Will fix that.

Reply:
BTW, this is the default value in flank

Reply:
Yeah, now I remember why I did it like this

https://github.com/runningcode/fladle/blob/7678e819b1b595b113c93e44d22a2b2cca4ca82d/buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt#L44

I just moved the default value

Reply:
Ah that makes sense. I like keeping all the default values here in one place. 👍